### DISCUSSION TITLE
Add the ability to use the mobile app for sudo mode and as a second factor for 2FA

### DISCUSSION THREAD

**Discussion by nihalgonsalves** [community/community#3881] [Mobile]
I use GitHub on my work computer, where I don't have my personal password manager account linked. It would be nice to be able to designate a device as a second factor and trusted device.

This would enable two things:

1. Approving logins via a push notification (similar to Okta, various banks, etc), with the usual TOTP, SMS, or hardware key as a second option.
2. Approving "sudo mode" via a push notification, for example when deleting repositories, editing branch protection rules, modifying access, etc.

**Comment by eliperkins** [comment]
@nihalgonsalves we hear ya! We'd love to see GitHub mobile become a second factor as well. Stay tuned. 👀
